From 40045c924fa3be046c73711b3089218ad52716c7 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Thu, 9 Oct 2014 21:00:13 -0400
Subject: [fs] xfs: negate xfs_icsb_init_counters error value

Message-id: <1412888441-4181-20-git-send-email-sandeen@redhat.com>
Patchwork-id: 97309
O-Subject: [RHEL7.1 PATCH 19/47] [fs] xfs: negate xfs_icsb_init_counters error value
Bugzilla: 1145837
RH-Acked-by: Brian Foster <bfoster@redhat.com>
RH-Acked-by: Dave Chinner <dchinner@redhat.com>

Bugzilla: 1145837
Upstream Status: Committed upstream
Build Info: https://brewweb.devel.redhat.com/taskinfo?taskID=8082426
Tested: locally using xfstests

Thanks,
-Eric

    commit bc147822d5ada188fff792691efffe89589e2e19
    Author: Dave Chinner <dchinner@redhat.com>
    Date:   Thu May 15 09:23:07 2014 +1000

    xfs: negate xfs_icsb_init_counters error value

    Signed-off-by: Dave Chinner <dchinner@redhat.com>
    Reviewed-by: Jie Liu <jeff.liu@oracle.com>
    Signed-off-by: Dave Chinner <david@fromorbit.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/xfs/xfs_super.c b/fs/xfs/xfs_super.c
index 0bb0095..ec83761 100644
--- a/fs/xfs/xfs_super.c
+++ b/fs/xfs/xfs_super.c
@@ -1436,7 +1436,7 @@ xfs_fs_fill_super(
  if (error)
   goto out_close_devices;
 
- error = xfs_icsb_init_counters(mp);
+ error = -xfs_icsb_init_counters(mp);
  if (error)
   goto out_destroy_workqueues;
 
-- 
1.7.1